CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ cmake_minimum_required(VERSION 3.1)
 ##
 project(nlohmann_json VERSION 3.9.1 LANGUAGES CXX)
 
+##
+## MAIN_PROJECT CHECK
```

Comment:
Please add documentation similar to https://github.com/nlohmann/json/pull/2514#issuecomment-743769382.